antora-ui-camel/src/js/02-fragment-jumper.js

```diff
@@ -1,6 +1,10 @@
 ;(function () {
   'use strict'
 
+  var shiftWindow = function () { window.scrollBy(0, -80) }
```

Comment:
These 80 pixels I understand they depend on the size of the bar. Is there no variable we can use here? Thinking on the future, when the bar size may change and this may not adjust properly anymore. I don't like having a long calculation each time this gets called, but it would be good to have some dependency on the height of the nav-bar.

Also, add some comments on the source code explaining what and why this. In a couple of months we will not remember why this was here.

Reply:
Sure, makes sense. I'm thinking I can find the height of the navbar and add some buffer to it for scrolling. Shouldn't be a long calculation.

Also, can you confirm if the rest of the code in the file is needed? I think it was meant to do this job, but I'm not sure if it serves any other purpose as well. Thanks!